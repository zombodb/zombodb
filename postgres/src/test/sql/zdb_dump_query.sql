SELECT * FROM zdb_dump_query('so_posts', 'id:*');
SELECT * FROM zdb_dump_query('so_users', 'beer');