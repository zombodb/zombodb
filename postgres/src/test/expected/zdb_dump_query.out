SELECT * FROM zdb_dump_query('so_posts', 'id:*');
   zdb_dump_query    
---------------------
 {                  +
   "match_all" : { }+
 }
(1 row)

SELECT * FROM zdb_dump_query('so_users', 'beer');
       zdb_dump_query        
-----------------------------
 {                          +
   "bool" : {               +
     "should" : [ {         +
       "term" : {           +
         "about_me" : "beer"+
       }                    +
     }, {                   +
       "term" : {           +
         "_all" : "beer"    +
       }                    +
     } ]                    +
   }                        +
 }
(1 row)

